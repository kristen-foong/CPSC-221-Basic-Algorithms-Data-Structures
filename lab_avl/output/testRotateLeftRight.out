~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[1mLeft-Right Rotation[22m
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
rotateRight
~[1mBefore[22m~
       5         
     /   \       
   1       8     
    \            
     3           

~[1mAfter insert([22m[1m2[22m[1m)[22m~
            __ 5 __              
         __/       \__           
       1               8         
         \                       
           3                     
          /                      
         2                       


