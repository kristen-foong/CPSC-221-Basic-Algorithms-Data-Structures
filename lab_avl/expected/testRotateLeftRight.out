~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[1mLeft-Right Rotation[22m
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
~[1mBefore[22m~
       5         
     /   \       
   1       8     
  / \            
 0   3           

~[1mAfter insert([22m[1m2[22m[1m)[22m~
rotateLeftRight
rotateLeft
rotateRight
       3         
     /   \       
   1       5     
  / \       \    
 0   2       8   


